lib/main_common.pm

```diff
@@ -2400,117 +2400,98 @@ sub load_virtualization_tests {
+        loadtest "virtualization/universal/prepare_guests";
+        loadtest "virtualization/universal/waitfor_guests";
+        if (check_var('PATCH_WITH_ZYPPER', 1)) {
+            loadtest "virtualization/universal/patch_and_reboot";
+            if (my $update_package = get_var('UPDATE_PACKAGE')) {
+                if ($update_package eq 'kernel-default') {
+                    loadtest "virt_autotest/login_console";
+                    loadtest "virtualization/universal/list_guests";
+                    loadtest "virtualization/universal/patch_guests";
+                } elsif ($update_package eq 'xen' || $update_package eq 'qemu') {
+                    loadtest "virt_autotest/login_console";
+                    loadtest "virtualization/universal/list_guests";
+                }
+            }
+        }
+        loadtest "virtualization/universal/kernel";
+        loadtest "virtualization/universal/finish";
+    }
+
+    if (get_var('ENABLE_SRIOV_NETWORK_CARD_PCI_PASSTHROUGH') && is_sle('>15')) {
+        loadtest "console/login";
+        loadtest "virt_autotest/cleanup_libvirtd_log";
+        loadtest "virt_autotest/sriov_network_card_pci_passthrough";
+        loadtest "virtualization/universal/finish";
+    }
+    my @test_cases = (
+        ['ENABLE_SAVE_AND_RESTORE', 'virtualization/universal/save_and_restore'],
+        ['ENABLE_GUEST_MANAGEMENT', 'virtualization/universal/guest_management'],
+        ['ENABLE_FINAL', 'virtualization/universal/ssh_final,virtualization/universal/virtmanager_final,virtualization/universal/smoketest,virtualization/universal/stresstest,console/perf'],
+        ['ENABLE_STORAGE', 'virtualization/universal/storage'],
+        ['ENABLE_WINDOWS', 'virtualization/universal/download_image,virtualization/universal/windows'],
+        ['ENABLE_DOM_METRICS', 'virtualization/universal/virsh_stop,virtualization/universal/xl_create,virtualization/universal/dom_install,virtualization/universal/dom_metrics,virtualization/universal/xl_stop,virtualization/universal/virsh_start', 'xen'],
+        ['ENABLE_IRQBALANCE', 'virt_autotest/xen_guest_irqbalance', 'xen'],
+        ['ENABLE_VIR_NET', 'virt_autotest/libvirt_host_bridge_virtual_network,virt_autotest/libvirt_nated_virtual_network,virt_autotest/libvirt_isolated_virtual_network'],
+        ['ENABLE_VIRTMANAGER', 'virtualization/universal/virtmanager_init,virtualization/universal/virtmanager_offon,virtualization/universal/virtmanager_add_devices,virtualization/universal/virtmanager_rm_devices'],
+        ['ENABLE_HOTPLUGGING', 'virtualization/universal/hotplugging_guest_preparation,virtualization/universal/hotplugging_network_interfaces,virtualization/universal/hotplugging_HDD,virtualization/universal/hotplugging_vCPUs,virtualization/universal/hotplugging_memory,virtualization/universal/hotplugging_cleanup'],
+        ['ENABLE_SNAPSHOTS', 'virt_autotest/virsh_internal_snapshot,virt_autotest/virsh_external_snapshot', 'kvm'],
```

Comment:
I think in this array, sriov test can just be handled the same way with other feature tests. We do not need to make them exclusive to each other. With switch for each of them, it is testsuite setting's responsibility to control which testsuite to have what feature tests.

Reply:
will add sriov test to this array， and deal with by swich.